tests/test_packages/test_connections/test_oef/test_communication.py

```diff
@@ -1093,10 +1093,11 @@ async def test_connecting_twice_is_ok(self, pytestconfig):
 async def test_cannot_connect_to_oef():
     """Test the case when we can't connect to the OEF."""
     oef_connection = _make_oef_connection(
-        address=FETCHAI_ADDRESS_ONE, oef_addr="a_fake_address", oef_port=10000,
+        address=FETCHAI_ADDRESS_ONE,
+        oef_addr="127.0.0.1",
+        oef_port=61234,  # use addr instead of hostname to avoid name resolution
```

Comment:
I believe the point was to have an invalid address, now we seem to have a valid one?

Reply:
maybe we could use mocking in this case